tests/dune

```diff
@@ -0,0 +1,11 @@
+(executable
+ (name test)
+ (libraries parany unix)
+)
+
+(alias
+ (name runtest)
+ (action
+  (run ./test.exe 6 25)
```

Comment:
Numbers here were completely arbitrary. I don't know if you have a better suggestion